cms/djangoapps/contentstore/models.py

```diff
@@ -21,3 +21,28 @@ def get_profile_whitelist(cls):
 
 class PushNotificationConfig(ConfigurationModel):
     """Configuration for mobile push notifications."""
+
+
+class MigrateVerifiedTrackCohortsSetting(ConfigurationModel):
+    """
+    ...
+    """
+    class Meta(object):
+        app_label = "contentstore"
+
+    old_course_id = TextField(
```

Comment:
Can do, I was basing this off of https://github.com/edx/edx-platform/blob/master/cms/djangoapps/contentstore/management/commands/export.py#L32 which I guess doesn't make sense if we'll be configuring this through Django Admin.